### PR TITLE
Added schema, registry and credential

### PR DESCRIPTION
**Pull request by Vikastc**
Did not test it , need your help in setting up the environment

### CODE REVIEW

**Review by Vikastc — COMMENTED** [review]
@amarts 
Type check and cleaning up of code

File: src/entity/Message.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    Column,
+    BeforeInsert,
+    CreateDateColumn,
+    UpdateDateColumn,
+    Unique,
+} from 'typeorm';
+import { v4 as uuidv4 } from 'uuid';
+
+@Entity()
+@Unique(['identifier', 'hash'])
```
Comment (amarts) [inline src/entity/Message.ts:13]: This should be removed.

File: src/entity/Schema.ts [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
 
-    @Column({ default: false })
-    registry?: boolean;
+  @Column()
+  description?: string;
 
-    @Column()
-    name?: string;
+  @Column()
+  Ischema?: string;
```
Comment (amarts) [inline src/entity/Schema.ts:26]: Ischema? Why capital `I`?

File: src/init.ts [hunk 3]
```diff
@@ -192,14 +193,15 @@ export async function setupDidAndIdentities() {
     } else {
       console.log("Failed to Create issuer DID");
     }
+    console.log("Donnnne");
```
Comment (amarts) [inline src/init.ts:196]: remove the log

**Review by amarts — APPROVED** [review]
minor comments. otherwise good to merge.

File: src/entity/Schema.ts (hunk 2 above)
Comment (Vikastc) [reply src/entity/Schema.ts:26]: Thought Ischema is better to read than ishema
If you want i can change it
@amarts

File: src/controller/credential_controller.ts [hunk 4]
```diff
@@ -52,72 +63,77 @@ export async function issueVC(
...
+    } catch (error) {
+      console.log("err: ", error);
+    }
   } catch (err: any) {
     console.log("Error: ", err);
   }
   // return document;
 
-  const url = `http://localhost:${PORT}/api/v1/message/:did`;
+  const url = `http://wallet:5001/api/v1/message/${holderDidUri}`;
```
Comment (amarts) [inline src/controller/credential_controller.ts:119]: as a next patch, send this as 'WALLET_URL' which is set from .env and added in docker-compose

**Review by amarts — APPROVED** [review]

File: src/controller/credential_controller.ts (hunk 4 above)
Comment (Vikastc) [reply src/controller/credential_controller.ts:119]: Sure

**Pull request merged**